src/main/java/org/saltyrtc/client/signaling/Signaling.java

```diff
@@ -466,28 +466,28 @@ public void onDisconnected(WebSocket websocket,
                             getLogger().warn("WebSocket closed, server is being shut down");
```

Comment:
This happens all the time. `.info`?

Reply:
Why would this happen all the time? By default the server should not just cut off the connection.

Reply:
After discussion: In Java we can probably leave this on warning level. In JS, it could have been triggered on client side by closing the tab...